frb_codegen/src/parser/ty.rs

```diff
@@ -1,45 +1,59 @@
 use std::collections::{HashMap, HashSet};
+
 use std::string::String;
 
+use itertools::Itertools;
+use quote::__private::Span;
```

Comment:
Hmm "private" seems to mean we should not use it. Maybe something like quote::Span?

Reply:
maybe `proc_macro2::Span`
I'm going to add the proc_macro2 dependency to Cargo.tom